mariadb/config.yaml

```diff
@@ -10,8 +10,8 @@ arch:
   - aarch64
   - amd64
   - i386
-backup_post: /bin/s6-svc -wd -d /run/s6/services/mariadb-lock/
-backup_pre: /bin/s6-svc -wU -u /run/s6/services/mariadb-lock/
+backup_post: s6-svc -wd -d /run/s6/services/mariadb-lock/
+backup_pre: s6-svc -wU -u /run/s6/services/mariadb-lock/
```

Comment:
```suggestion
backup_post: s6-svc -wd -d /run/services/mariadb-lock/
backup_pre: s6-svc -wU -u /run/services/mariadb-lock/
```

Reply:
See my comment below. `/run/s6/services/mariadb-lock/` and `/run/services/mariadb-lock/` both resulted in error. Only `/run/s6/legacy-services/mariadb-lock/` seemed to work.